client/login.py

```diff
@@ -0,0 +1,32 @@
+import pygame as pg
+
+pg.init()
+COLOR = pg.Color('red')
+
+class TextBox:
```

Comment:
this class should actually inherit from a pygame rect, it will simplify your code a lot more!